torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1083,12 +1083,44 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
              const ExprHandle& weight) { return a + weight * (end - a); });
     } break;
     case aten::remainder: {
-      return computeTwoOperand(
-          "aten_remainder",
-          v,
-          [](const ExprHandle& lhs, const ExprHandle& rhs) {
-            return fmod((rhs + fmod(lhs, rhs)), rhs);
-          });
+      auto imodImpl = [](const ExprHandle& lhs, const ExprHandle& rhs) {
+        return Mod::make(lhs, rhs);
```

Comment:
Is `remainder` always equivalent to `mod`? I thought there was a difference when some of the operands were negative.

Reply:
afaik mod is equivalent to remainder for integers.